types/mongoose/index.d.ts

```diff
@@ -2993,7 +2994,8 @@ declare module "mongoose" {
     translateAliases(raw: any): any;
 
     /**
-     * Sends multiple insertOne, updateOne, updateMany, replaceOne, deleteOne, and/or deleteMany operations to the MongoDB server in one command. This is faster than sending multiple independent operations (like) if you use create()) because with bulkWrite() there is only one round trip to MongoDB.
```

Comment:
This change looks like a typo - which I don't think you want (maybe a merge conflict resolution?)  - otherwise this looks good and I would have just merged it right now 👍

Reply:
yep it was